ClientLibrary/lib/src/main/java/com/microsoft/projectoxford/vision/VisionServiceRestClient.java

```diff
@@ -304,4 +305,8 @@ private void AppendParams(Map<String, Object> params, String name, String[] args
             params.put(name, features);
         }
     }
+
+    private static String getApiUrlFromRegion(String region) {
+        return String.format(DEFAULT_API_ROOT, region).replaceAll("/$", "");
```

Comment:
The `replaceAll` is superfluous since the constant doesn't end with a slash.  Please remove.

Reply:
Fixed!